pldoc/static/sass/_components.scss

```diff
@@ -498,9 +723,49 @@
     }
 }
 
+// ui depth
+.pldoc-element-depth {
+
+    // padding for demo purposes
+    .depth {
+        padding: spacing-vertical(small) spacing-horizontal(base);
+    }
+
+    // light text on darker backgrounds for demo purposes
+    .depth--2,
```

Comment:
Oh, this isn't BEMing or anything, this class (a product of the [ui-depth](https://github.com/edx/ux-pattern-library/blob/master/pattern-library%2Fsass%2Fpatterns%2F_depth.scss#L92) pattern) is just signifying a UI element that has a depth of negative 2. If I could, I'd write it as ``.depth-(-2)``, but that's not valid CSS.

Make sense?